cmake/BundledCivetWeb.cmake

```diff
@@ -64,12 +61,11 @@ function(use_bundled_civetweb SOURCE_DIR BINARY_DIR)
     # Build project
     ExternalProject_Add(
             civetweb-external
-            URL "https://github.com/civetweb/civetweb/archive/v1.12.tar.gz"
-            URL_HASH "SHA256=8cab1e2ad8fb3e2e81fed0b2321a5afbd7269a644c44ed4c3607e0a212c6d9e1"
+            GIT_REPOSITORY "https://github.com/civetweb/civetweb.git"
+            GIT_TAG "4447b6501d5c568b4c6c0940eac801ec690b2250" # commit containing fix for MSVC issue https://github.com/civetweb/civetweb/issues/1024
```

Comment:
I would prefer to stay on HTTP and a release version, and keep the necessary patch.
Switching to FetchContent and keeping the git commit ref is a reasonable alternative, too. It doesn't make the commit reference any more stable and tested, but at least it helps with caching artifacts between builds, since one can specify an external FETCHCONTENT_BASE_DIR.